src/installLogsPrinter.d.ts

```diff
@@ -69,7 +75,7 @@ declare namespace installLogsPrinter {
       string,
       | 'json'
       | 'txt'
-      | ((allMessages: AllMessages) => void)
+      | CustomOutputProcessorCallback
```

Comment:
Essentially just adding the `this: CustomOutputProcessor` to this callback, to improve types. Let's someone know what's available in the `this.`